app/models/API1812/latePaymentPenalty/LatePaymentPenalty.scala

```diff
@@ -28,8 +31,33 @@ case class LatePaymentPenalty(principalChargeReference: String,
                               LPP1HRPercentage: Option[Double],
                               LPP2Days: Option[String],
                               LPP2Percentage: Option[Double],
-                              penaltyChargeReference: Option[String])
+                              penaltyChargeReference: Option[String],
+                              timeToPay: Option[Seq[TimeToPay]]) {
+
+  def hasTimeToPay(implicit appConfig: AppConfig): Boolean = timeToPay.fold(false)(_.exists { ttp =>
+    (!ttp.TTPStartDate.isAfter(DateService.now)) &&
```

Comment:
I appreciate this is not your task but would you mind updating the breathing space function to have the same logic as written here? I think it's more succinct, and that way it keeps our logic consistent. It should be a quick change and the BS tests will validate that behaviour hasn't changed.

You could also remove the redundant parentheses surrounding each statement either side of the &&